Source/Analytics/Web/src/assets/main.scss

```diff
@@ -3,3 +3,28 @@
 html {
     font-size: 16px;
 }
+
+body {
+    margin: 0;
+}
+
+h1 {
```

Comment:
I think we should still use classes and not set styling directly on the html tags. It will quickly and quite easily become a need for things to be a bit different.

Reply:
I agree – I took out the area-specific styling and added them to classes.